standalone/README.md

```diff
@@ -19,36 +19,135 @@ In order to build and use the PSENscan Standalone C++ Library you need Ubuntu ve
 
 Make sure you have all necessary tools and libraries installed:
 ```
-sudo apt-get install build-essential cmake git libboost-system-dev libconsole-bridge-dev libfmt-dev
+sudo apt-get install build-essential cmake git libboost-all-dev libconsole-bridge-dev libfmt-dev
 ```
 
-## Get Started
-First, clone this repository:
-```
-git clone https://github.com/PilzDE/psen_scan_v2.git
-```
+## Get Started on Linux
 
-Next, navigate to the standalone folder and create a `build` directory.
-```
-cd psen_scan_v2/standalone/
-mkdir build
-```
+- First, clone this repository:
+  ```
+  git clone https://github.com/PilzDE/psen_scan_v2.git
+  ```
 
-Lastly, execute the following:
-```
-cd build/ && cmake .. && make
-```
+- Next, navigate to the standalone folder and create a `build` directory.
+  ```
+  cd psen_scan_v2/standalone/
+  mkdir build
+  ```
+
+- Lastly, execute the following:
+  ```
+  cd build/ && cmake .. && make
+  ```
 
 The library should now be built and available under `psen_scan_v2/standalone/build/libpsen_scan_v2_standalone.a`
 
-## Usage example
+### Usage example
 An example application, which prints distance data to the screen, is built by default and can be executed in the `build` folder:
 ```
 ./psen_scan_v2_standalone_app
 ```
 
+## Get Started on Windows
+### Dependencies
+- Visual Studio
+- Boost (https://www.boost.org/)
+- console_bridge https://github.com/ros/console_bridge
+- fmt (https://github.com/fmtlib/fmt)
+
+### Build and install dependencies
+#### Boost
```

Comment:
To be consistant with the enumeration above: First paragraph should be Visual Studio?!